cfme/tests/cloud/test_snapshot.py

```diff
@@ -165,3 +132,30 @@ def test_snapshot_cloud_tenant():
         1685300
     """
     pass
+
+
+@pytest.mark.manual
+@pytest.mark.provider([OpenStackProvider])
+@test_requirements.snapshot
```

Comment:
All tests in this module have the same requirement, so please put it for the whole module

`pytestmark = [test_requirements.snapshot]`

Reply:
@lina-nikiforova some have `rbac` and `auth`

Reply:
Oh but they are different files, please ignore above :man_facepalming: